src/apps/settings/src/app/views/Robots/Robots.js

```diff
@@ -20,6 +20,7 @@ import styles from "./Robots.less";
 
 export default connect(state => {
   return {
+    domain: state.instance.domain,
```

Comment:
This should be removed right? Because we are using the `useDomain` hook vs this value.